lib/org/chromium/webidl/ast/Attribute.js

```diff
@@ -13,7 +13,18 @@ foam.CLASS({
     'org.chromium.webidl.ast.Typed',
   ],
 
+  imports: [ 'source? as ctxSource' ],
+
+  properties: [
+    {
+      name: 'source',
+    },
+  ],
+
   methods: [
+    function init() {
+      this.source = this.ctxSource;;
```

Comment:
Also, please add `this.SUPER()` to the end of the method to ensure other `init()` work runs.